plugins/services/src/js/components/__tests__/ServiceStatusProgressBar-test.js

```diff
@@ -17,11 +17,14 @@ const testData = {
 };
 
 const testInstanceCount = 4;
-describe("#HealthBar", function() {
+describe("#ServiceStatusProgressBar", function() {
   beforeEach(function() {
     this.container = global.document.createElement("div");
     this.instance = ReactDOM.render(
-      <HealthBar tasksSummary={testData} instancesCount={testInstanceCount} />,
+      <ServiceStatusProgressBar
+        tasksSummary={testData}
```

Comment:
This component only accepts one prop `service` :)

Reply:
Yes, this PR is a WIP.